app/controllers/jobs_controller.rb

```diff
@@ -1,16 +1,23 @@
 class JobsController < ApplicationController
+  before_action :find_job, only: [:show, :edit, :update, :destroy]
+  before_action :find_company, only: [:new, :create, :edit, :update, :destroy]
+
   def index
-    @company = Company.find(params[:company_id])
-    @jobs = @company.jobs
+    if interest_params.nil?
+      @company = Company.find(params[:company_id])
+      @contacts = Contact.where(company: @company)
+      @jobs = @company.jobs
+    else
+      @jobs = Job.order(interest_params).reverse
```

Comment:
ordering the jobs and then reversing them shouldn't happen in the controller. i suggest writing a method in your `Job` that accomplishes this